docs/tutorials/docker-integration.md

```diff
@@ -0,0 +1,161 @@
+# Docker integration
```

Comment:
WDYT about adding a header with you as @author just like other tutorials?

Reply:
Added in [8783e99](https://github.com/emacs-lsp/lsp-mode/pull/3448/commits/8783e99dadb8d0a9212b988f540f88f1f471da18)